.github/templates/linux_ci_workflow.yml.in

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Chown workspace
         run: |
           # Ensure the working directory gets chowned back to the current user
-          docker run --rm -v "$(pwd)":/v -w /v alpine chown -R "$(id -u):$(id -g)" .
+          docker run --rm -v "$(pwd)":/v -w /v "${ALPINE_IMAGE}" chown -R "$(id -u):$(id -g)" .
```

Comment:
ALPINE_IMAGE was defined at the beginning of the file

Reply:
could you comment on the `env:` section where ALPINE_IMAGE were defined?

Reply:
@samestep @walterddr Of course, will unblock first and save a new run of CI and do the comments in the following PRs